### PR TITLE
Added responsive images and input component

### PR DESCRIPTION
**Pull request by prakashsakari**
Hi kindly review my PR, for images and input

### CODE REVIEW

File: Image/image.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Prakash Sakari">
+    <title>Responsive Images</title>
+    <link rel="stylesheet" href="image.css">
+</head>
+<body>
+    <div class="container-img-responsive">
+        <img class="img-responsive" src="/assets/shoes.jpg" alt="image">
```
Comment (Aditi0505) [inline Image/image.html]: Suggestion: Can add a better alt name. It is a given that it is a image. You can describe what image it is. In this case - shoes.

**Review by Aditi0505 — APPROVED** [review]
Looks good to me

File: Input/input.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.form-label {
+    display: block;
+    padding: 1rem;
+    text-decoration-line: underline;
+    font-weight: 500;
+}
+
+.lh-ls {
```
Comment (rohanmathur91) [inline Input/input.css:24]:
``` suggestion
this does not give an idea about which properties this class will apply, try to give some other class name.
```

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

**Pull request merged**